consensus/hotstuff/voteaggregator/vote_aggregator_v2_test.go

```diff
@@ -67,11 +71,17 @@ func (s *VoteAggregatorV2TestSuite) SetupTest() {
 	s.aggregator, err = NewVoteAggregatorV2(unittest.Logger(), s.notifier, 0, s.collectors)
 	require.NoError(s.T(), err)
 
+	ctx, cancel := context.WithCancel(context.Background())
+	s.cancel = cancel
+	signalerCtx, _ := irrecoverable.WithSignaler(ctx)
+
 	// startup aggregator
+	s.aggregator.Start(signalerCtx)
 	<-s.aggregator.Ready()
```

Comment:
In the mindeset of "fail early": 
* this could have a time-out, to fail the test, if component startup hangs?